0400-results.Rmd

```diff
@@ -6,6 +6,20 @@ A map showing sample sites for fish sampling, fish habitat assessments and disch
 knitr::include_graphics("fig/elk_fish_passage_2022.png")
 ```
 
+**Habitat Model Parameters** 
```

Comment:
We have scripts that do this already.... Is this coming from the map screenshot?  I was thinking that you would insert that jpg onto the map like simon does.....

Reply:
It was from the screenshot yes. You can take that text out then. I tried putting the screenshot in the legend but the text was too small and not legible and it looked bad to be honest.

Reply:
I don't like the tone of this reply

Reply:
What? That wasn't meant to be an annoyed tone. I was unsure whether or not to put that text in so I was suggesting that you could take it out if we already have it somewhere else. Just being honest.